packages/core/src/components/callout/callout.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+
+import { Classes, IIntentProps, IProps } from "../../common";
+import { IconName } from "../icon/icon";
+
+/** This component also supports the full range of HTML `<div>` props. */
+export interface ICalloutProps extends IIntentProps, IProps {
+    /** Name of icon to render on left-hand side. */
```

Comment:
nit: We're inconsistent about including articles like `a`, `an`, and `the`. I rather like them.

Reply:
forgotten articles? sounds like FAKE NEWS!